src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssStyleObject = {};
+  const cssStyles = sourceString.split(';');
```

Comment:
Consider trimming the `sourceString` before splitting it to handle any leading or trailing whitespace that might affect the splitting process.